lib/Styles/Common/_controls.scss

```
@@ -0,0 +1,10 @@
+@mixin control {
+    @extend %outset-background;
+    background-color: bg-color(0);
+    background-clip: border-box;
+
+    @include border-interactive;
+
+    transition: all duration("in-place") easing("default");
```

Let's use easing function from animate?
```suggestion
    transition: all duration("in-place") $easing;
```

---

It does use the function from `_animate`. `$easing` isn't defined outside the easing function

---

Sorry I don't know how I misread this lmao